generated/nidcpower/nidcpower/_grpc_stub_interpreter.py

```diff
@@ -80,7 +80,10 @@ def self_cal(self, channel_name):  # noqa: N802
         )
 
     def clear_latched_output_cutoff_state(self, channel_name, output_cutoff_reason):  # noqa: N802
-        raise NotImplementedError('clear_latched_output_cutoff_state is not supported over gRPC')
+        self._invoke(
+            self._client.ClearLatchedOutputCutoffState,
+            grpc_types.ClearLatchedOutputCutoffStateRequest(vi=self._vi, channel_name=channel_name, output_cutoff_reason=output_cutoff_reason.value),
```

Comment:
Sorry to do this to you again, but if you could grab d288 I would appreciate it. A change I made later yesterday made it so it passes both the raw and enum as a oneof, and now our code needs to set the `output_cutoff_reason_raw` field.

Reply:
I've since updated to d305.